python/mooseutils/PostprocessorReader.py

```diff
@@ -0,0 +1,52 @@
+from MooseDataFrame import MooseDataFrame
+import tools
+
+class PostprocessorReader(MooseDataFrame):
+    """
+    An extension to the MooseDataFrame to provide functionality mirroring that of the VectorPostprocessorReader.
+
+    Args:
+        filename[str]: The csv file to read.
+    """
+
+    def __init__(self, filename, **kwargs):
+        super(PostprocessorReader, self).__init__(filename, **kwargs)
+
+    def __call__(self, keys, time=None, warning=True, **kwargs):
+        """
+
```

Comment:
OK, looking a few lines down, it uses `self.data[keys]`. It seems a bit unusual to index with a list but maybe the `pandas.DataFrame()` supports that.